README.md

```diff
@@ -51,22 +60,22 @@ For example, we can analyse all contracts labelled with type `reentrancy` with t
 python3 smartBugs.py --tool oyente --file dataset/reentrancy
 ```
 
-By default, results will be placed in the directory results. 
-The results of the analysis on 47,587 contracts is available here: https://github.com/smartbugs/smartbugs-wild
+By default, results will be placed in the directory `results`. 
 
 ## Known Limitations
 
 When running a tool the user must be aware of the solc compatibility. Due to the major changes introduced in solidity v0.5.0, we provide the option to pass another docker image to run contracts with solidity version below v0.5.0. However, please note that there may still be problems with the solidity compiler when compiling older versions of solidity code. 
 
 ## Smart Contracts dataset
 
-We propose two dataset a smart contract dataset with SmartBugs.
-The first dataset contains 69 annotated contracts to evaluate the accuracy of analysis tools.
-The second dataset contains 47,518 unique contract from the Ethereum network.
+We make available two smart contract datasets with SmartBugs:
+- **SB Curated**: a curated dataset with 69 annotated contracts that can be used to evaluate the accuracy of analysis tools    .
+- **SB Wild**: a dataset with 47,518 unique contract from the Ethereum network (for details on 3 how they were collected, please see [the ICSE 2020 paper](https://arxiv.org/abs/1910.10601)).
+
 
 ### SB Curated
 
-SB Curated provides a collection of vulnerable Solidity smart contracts organized according to the [DASP taxonomy](https://dasp.co). It is available in the `dataset` repository.
+[SB Curated](https://github.com/smartbugs/smartbugs/blob/master/dataset) provides a collection of vulnerable Solidity smart contracts organized according to the [DASP taxonomy](https://dasp.co). It is available in the `dataset` repository.
 
```

Comment:
link to `dataset` ?

Reply:
Yes, that's the directory of the curated dataset.